packages/docs/next.config.mjs

```diff
@@ -35,13 +35,15 @@ const nextConfig = {
   basePath: process.env.DOCS_BASE_URL || '',
   experimental: {
     externalDir: true,
+    esmExternals: 'loose',
   },
   images: {
     dangerouslyAllowSVG: true,
     contentDispositionType: 'attachment',
     contentSecurityPolicy: "default-src 'self'; script-src 'none'; sandbox;",
   },
   trailingSlash: true,
+  transpilePackages: ['@fuels/connectors', '@fuels/react'],
```

Comment:
do we need it? these packages are not internals

Reply:
yes we need it due to build problems